massa-client/src/repl.rs

```diff
@@ -135,20 +136,39 @@ impl Completion for CommandCompletion {
     /// Simple completion implementation based on substring
     fn get(&self, input: &str) -> Option<String> {
         let input = input.to_string();
-        let suggestions: Vec<&str> = self
-            .options
-            .iter()
-            .filter(|s| s.len() >= input.len() && input == s[..input.len()])
-            .map(|s| &s[..])
-            .collect();
-        if !suggestions.is_empty() {
-            println!();
-            for suggestion in &suggestions {
-                println!("{}", style(suggestion).dim());
+        if input.contains(' ') {
+            let mut args: Vec<&str> = input.split(' ').collect();
+            let mut default_path = "./";
+            let path_to_complete = args.last_mut().unwrap_or_else(|| &mut default_path);
+            let suggestions: Vec<String> = glob(&(path_to_complete.to_owned() + "*"))
```

Comment:
I see many unwraps here, in which cases would the client crash ?

Reply:
We don't want the client to crash when the user inputs/does invalid stuff, it makes for terrible UX.
Let's just to tell them why the command failed and stay in the REPL.
More generally, we need to be *very* careful when using things that might panic. And a good practice is to leave a comment besides every except/unwrap etc... to explain why it won't happen, and if it happens, why it is absolutely unrecoverable anyways

Reply:
Yes I have changed to handles the crash already I just didn't propagate the error to the client but will add it !